samza-core/src/main/java/org/apache/samza/clustermanager/ContainerProcessManager.java

```diff
@@ -612,4 +613,14 @@ private String getPendingProcessorId(String containerId) {
   private void handleContainerStop(String processorId, String containerId, String preferredHost, int exitStatus, Duration preferredHostRetryDelay) {
     containerManager.handleContainerStop(processorId, containerId, preferredHost, exitStatus, preferredHostRetryDelay, containerAllocator);
   }
+
+  public Map<String, SamzaResource> getCurrentRunningContainers() {
+    return state.runningProcessors;
+  }
+
+  // processorId 0,1,2 = current yarn container id not samza container id
+  // preferredHost = fully qualified hostname
+  public void requestMoveContainer(String processorId, String preferredHost) {
```

Comment:
containerPlacement vs. moveContainer vs. ContainerMovement 
pick one and use throughout this PR